File: Week3/src3/numbers.c
// Implements linear search for numbers

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // An array of numbers
    int numbers[] = {4, 6, 8, 2, 7, 5, 0};

    // Search for 0
    for (int i = 0; i < 7; i++)
    {
        if (numbers[i] == 0)
        {
            printf("Found\n");
            return 0;
        }
    }
    printf("Not found\n");
    return 1;
}

File: Week6/src6/3/numbers.py
# Implements linear search for numbers

import sys

# A list of numbers
numbers = [4, 6, 8, 2, 7, 5, 0]

# Search for 0
if 0 in numbers:
    print("Found")
    sys.exit(0)

print("Not found")
sys.exit(1)
